client/rpc/src/eth.rs

```diff
@@ -263,7 +263,7 @@ impl<B, C, P, CT, BE, H: ExHashT> EthApiT for EthApi<B, C, P, CT, BE, H> where
 	fn syncing(&self) -> Result<SyncStatus> {
 		if self.network.is_major_syncing() {
 			let block_number = U256::from(
-				self.client.info().best_number.clone().unique_saturated_into()
+				UniqueSaturatedInto::<u128>::unique_saturated_into(self.client.info().best_number.clone())
```

Comment:
Curious why is this change? Does the type inference of `unique_saturated_into` fail somehow?

Reply:
with the latest rustc version toolchain (rustc 1.49.0 (e1884a8e3 2020-12-29)), this compiling issue will happen.

Please check https://github.com/paritytech/frontier/runs/1642051460 for more information.